### PR TITLE
Add registration retries

### PR DESCRIPTION
**Pull request by matthewloring**

### CODE REVIEW

**Comment by matthewloring** [comment]
@ofrobots PTAL.

File: lib/debuglet.js [hunk 1]
```diff
@@ -142,8 +142,8 @@ Debuglet.prototype.scheduleRegistration_ = function(seconds) {
 
   function onError(err) {
     that.logger_.error('Failed to re-register debuggee: ' + err);
-    that.logger_.error('Disabling gcloud debuglet');
-    that.stop(); // fatal error
+    that.scheduleRegistration_(Math.min((seconds + 1) * 2,
```
Comment (ofrobots) [inline lib/debuglet.js]: Let's just make this simpler and omit the exponential backoff. The default value of 40 is small enough. WDYT?

Comment (ofrobots) [inline lib/debuglet.js]: If you agree, let's rename to name of the config property and drop the `max`.

**Comment by ofrobots** [comment]
LGTM w/ comment.

File: lib/debuglet.js (hunk 1 above)
Comment (matthewloring) [inline lib/debuglet.js]: I don't think there is a substantial complexity difference so lets keep the backoff.

**Pull request merged**